src/dialer/queue.js

```diff
@@ -236,13 +244,15 @@ class Queue {
 
     // If we're not muxed yet, add listeners
     connectionFSM.once('muxed', () => {
+      this.blackListCount = 0 // reset blacklisting on good connections
```

Comment:
Should `this.blackListed` also be reset?

Reply:
If we're able to dial the peer it should already be null, since `this.blackListed ` is checked before any dial attempt is made. Setting this to 0 ensures that if we weren't able to previously dial the peer that we clear the blacklist so we don't end up waiting a day to dial if it has some intermittent availability.